lkmeans/clustering/unsupervised.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+from copy import deepcopy
+
+import numpy as np
+from numpy.typing import NDArray
+
+from lkmeans.clustering.base import Clustering
+from lkmeans.clustering.utils import assign_to_cluster, calculate_inertia, set_type
+
+
+def init_centroids(data: NDArray, n_clusters: int) -> NDArray:
+    indices = np.random.choice(data.shape[0], n_clusters, replace=False)
+    centroids = data[indices]
+    return centroids
```

Comment:
Init centroids for unsupervised must be placed near assing_to_clusters unsupervised